config/routes.rb

```diff
@@ -21,7 +21,7 @@
     get 'support'
     get 'terms'
     get 'dashboard'
-    get 'site-changes'
+    get 'site_changes'
```

Comment:
If you change this to `get 'site-changes', to: :site_changes` you can have the URL "/site-changes" which I believe is preferable.